packages/hdwallet-native/src/native.ts

```diff
@@ -210,6 +210,7 @@ export class NativeHDWallet
   readonly _supportsBTC = true;
   readonly _supportsETH = true;
   readonly _supportsCosmos = true;
+  readonly _supportsEthSwitchChain = true;
```

Comment:
i don't think this is true yet until we impl avalanche support

Reply:
Removed in b3852ba8a44e80c11c1a5f2804dbcf687d3b4116

Reply:
It will probably never be true either way as we don't switch chains per se with native